spec/fixtures/vcr_cassettes/association/belongs_to_many.yml

```diff
@@ -0,0 +1,439 @@
+        os macOS/16;
+      Authorization:
+      - Bearer 60229df5876f14f499870d0d26f37b1323764ed637289353a5f74a5ea190c214
+      Content-Type:
+      - application/vnd.contentful.delivery.v1+json
+      Accept-Encoding:
+      - gzip
+      Connection:
+      - close
+      Host:
+      - cdn.contentful.com
+      User-Agent:
+      - http.rb/2.2.2
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Access-Control-Allow-Headers:
+      - Accept,Accept-Language,Authorization,Cache-Control,Content-Length,Content-Range,Content-Type,DNT,Destination,Expires,If-Match,If-Modified-Since,If-None-Match,Keep-Alive,Last-Modified,Origin,Pragma,Range,User-Agent,X-Http-Method-Override,X-Mx-ReqToken,X-Requested-With,X-Contentful-Version,X-Contentful-Content-Type,X-Contentful-Organization,X-Contentful-Skip-Transformation,X-Contentful-User-Agent,X-Contentful-Enable-Alpha-Feature
+      Access-Control-Allow-Methods:
+      - GET,HEAD,OPTIONS
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Expose-Headers:
+      - Etag
+      Access-Control-Max-Age:
+      - '86400'
+      Cache-Control:
+      - max-age=0
+      Content-Encoding:
+      - gzip
+      Content-Type:
+      - application/vnd.contentful.delivery.v1+json
+      Etag:
+      - W/"e1e85d233f7220aa48a546d1d57e2995"
+      Server:
+      - Contentful
+      X-Content-Type-Options:
+      - nosniff
+      X-Contentful-Request-Id:
+      - b83300741b5c566f0dbb05becfd5f1cf
+      Content-Length:
+      - '719'
+      Accept-Ranges:
+      - bytes
+      Date:
+      - Wed, 25 Apr 2018 11:33:37 GMT
+      Via:
+      - 1.1 varnish
+      Age:
+      - '720'
+      Connection:
+      - close
+      X-Served-By:
+      - cache-fra19151-FRA
+      X-Cache:
+      - HIT
+      X-Cache-Hits:
+      - '1'
+      X-Timer:
+      - S1524656018.681447,VS0,VE1
+      Vary:
+      - Accept-Encoding
+    body:
+      encoding: ASCII-8BIT
+      string: !binary |-
+        H4sIAAAAAAAAA+1YS0/jMBC+91dEPi9R4qalzY1FrPaw7K5ET4t6cBMDVhMnxKZSF/Hf1484saO0BBYhQetDVT9mxvP6Mp7HkecBtmUg9h7FXzHh2xKLGTirKrQFYu3pizzDC44ysT5RM7YmpZgEapKRnHAxC4NALxCOc8nxWnHUfDtilChWokTKMif0onUZuaAWzaV+ELoGUmg7hHi6XtSXvlIcOwdIKvVBEBbw/jYfT6aJVMuMp+a/UlQPoGnuELtEdLvAjF8WKc4szs2VzgvKMeXqBhaDpMKI4/RMGgbAIJydBNEJjBZhFEdhHE79eRT8sQkeyvRlBJhuSFXQXAgfZEOtUo4Yx1XXREPte2HJfNaGFd4QRgoqQ6M+3JgYpISVGdp+IzhT7qEob+2nZ2L1O2Ke9IAnXeC5PgApZklFSq5FgEYncCOZtgEoXeqEmLaEI1GeaaT+tO+idox9rrb5qrDDQIZ/kaCM/MVSjRuUMWyHH6jw/QOpdmwKI6BVtmOzEFklIshw7QvTHq24iVXm+LjRrTVk54Cb944O76ShI9NgiK3iM0CwEW4QOSTCwfW9TmmXkfKqRA47fQ1iaQLzCxqT2rChd5cOFAmsdObLPUh1QXml4NUMC4fqVc28TpkPjKKt/Sx8NOH2cvgMwjgK/GA2HQqf/QSHAJ9H1GxR85Ml1QpnBb1li0J+Hn+jSlYB/5tek0UYxjCIw1P/FM4GpNcegkNIr697fHCsTvpqruSOZKmI1f7apH/3WJiIElA/pTqFiYMB59K0xwrlVe+8HkO+CZaGc/HY86dwPhRL+wkODkt1LBsXHCyUfrKSRXRTflH8xs0UUdxD6EfTcECKqe5LAOPx2J9F0CH4SCkG67h4TTNFOODYS8Htq6CF+X29lP5yped1JT/Qu5uV79Uqcps3Vnu003QwLYe60SCsshw9jf4BVmWKqBMWAAA=
```

Comment:
:thinking: Can you make this not base encoded? For testing, it would be really nice to see what you are actually feeding in ...

Reply:
Answer: no.